targets/nuttx-stm32f4/jerry_main.c

```diff
@@ -558,6 +558,26 @@ jerry_port_release_source (uint8_t *buffer_p) /**< buffer to free */
   free (buffer_p);
 } /* jerry_port_release_source */
 
+/**
+ * Normalize a file path
+ *
+ * @return length of the path written to the output buffer
+ */
+size_t
+jerry_port_normalize_path (const char *in_path_p, /**< input file path */
```

Comment:
Please add the same to `targets/tizenrt-artik053/apps/jerryscript/jerry_main.c`.

Reply:
Done.